src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -492,6 +496,7 @@ impl VsockMuxer {
 
     /// Remove a connection from the active connection poll.
     fn remove_connection(&mut self, key: ConnMapKey) {
+        METRICS.vsock.conns_removed.inc();
```

Comment:
A similar comment here, `key` could not be present in the `conn_map`, can we move this someplace where we are sure a connection has been indeed removed (in the `if` block should be ok).

Reply:
Done